src/App.js

```diff
@@ -129,6 +171,28 @@ class App extends Component {
           <div className="Title">
             { calculatorInfo && calculatorInfo.title}
           </div>
+          <div className="Content Stack">
```

Comment:
I'm not great at styling front-end stuff, so I just used the same class names as what you had for calculating exclusion dates